.gitignore

```diff
@@ -1,7 +1,7 @@
 node_modules
 dist
 ConvivaIntegration/out
-demo/components/helper
+demo/components/conviva
```

Comment:
They are not ignored 🤔 Please check why. You might need to clean the git cache before changing the `.gitignore`. There is something like that.